tests/test_obs.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+import pytest
+
+from ecgtools.parsers.observations import parse_amwg_obs
+
+df = pd.DataFrame()
+
+
+@pytest.mark.parametrize(
+    'file_path',
+    [
+        '/glade/p/cesm/amwg/amwg_diagnostics/obs_data/AIRS_01_climo.nc',
+        '/glade/p/cesm/amwg/amwg_diagnostics/obs_data/MODIS_ANN_climo.nc',
```

Comment:
Could we add tiny samples (1-2 MB) of these files to the `sample_data` directory within this repository so as to enable running tests without GLADE access dependency?